coconut/control/control.go

```diff
@@ -710,28 +702,23 @@ func RefreshRepos(cxt context.Context, rpc *coconut.RpcClient, cmd *cobra.Comman
 		return err
 	}
 
-	var response *pb.RefreshReposReply
 	if len(args) == 0 {
-		response, err = rpc.RefreshRepos(cxt, &pb.RefreshReposRequest{Index: -1}, grpc.EmptyCallOption{})
+		_, err = rpc.RefreshRepos(cxt, &pb.RefreshReposRequest{Index: -1}, grpc.EmptyCallOption{})
 	} else if len(args) == 1 {
 		index, _ := strconv.ParseInt(args[0], 10, 32)
 
-		response, err = rpc.RefreshRepos(cxt, &pb.RefreshReposRequest{Index: int32(index)}, grpc.EmptyCallOption{})
+		_, err = rpc.RefreshRepos(cxt, &pb.RefreshReposRequest{Index: int32(index)}, grpc.EmptyCallOption{})
 	}
 
 	if err != nil {
+		fmt.Fprintln(o, "Repository refresh operation failed.")
 		return err
 	}
 
-	errorString := response.GetErrorString()
-	if errorString == "" {
-		if len(args) == 0 {
-			fmt.Fprintln(o, "Repositories refreshed succesfully")
-		} else {
-			fmt.Fprintln(o, "Repository refreshed succesfully")
-		}
+	if len(args) == 0 {
+		fmt.Fprintln(o, "Repositories refreshed succesfully")
```

Comment:
Both if & else condition are doing the same thing. Is the If condition still needed?

Reply:
One is "repositories" and the other is "repository".